src/containers/my-quizzes/create-or-edit-quiz-container/CreateOrEditQuizContainer.tsx

```diff
@@ -51,12 +55,16 @@ const CreateOrEditQuizContainer = ({
   const { setAlert } = useSnackBarContext()
   const { openModal } = useModalContext()
   const navigate = useNavigate()
+  const { id } = useParams()
   const [isCreationOpen, setIsCreationOpen] = useState<boolean>(false)
+  const [category, setCategory] = useState<Category | null>(null)
```

Comment:
should this state be defined here, because if I choose the category and change tab, then, when I return, this state will be null again

Reply:
It is just for now, because the category dropdown will be created later.